src/runtime/account.ts

```diff
@@ -64,6 +83,64 @@ export class Account {
     await this.keyStore.setKey(this.networkId, accountId, keyPair);
   }
 
+  async addKey(accountId: string, keyPair?: KeyPair): Promise<PublicKey> {
+    let pubKey: PublicKey;
+    if (keyPair) {
+      const key = await nearAPI.InMemorySigner.fromKeyPair(
+        this.networkId,
+        accountId,
+        keyPair
+      );
+      pubKey = await key.getPublicKey();
+    } else {
+      pubKey = await this.signer.createKey(accountId, this.networkId);
+    }
+    return pubKey;
+  }
+
+  async createAccount(
+    accountId: string,
+    { keyPair, initialBalance }: { keyPair?: KeyPair; initialBalance: string }
```

Comment:
Should this provide a default `initialBalance`?

Reply:
Ah good point. We had been using the config passed to the runtime, but perhaps with a refactor and passing the runtime to the account, we'll have access of that here.